### PR TITLE
Weather 2 wind & storm affecting ships

### PR DESCRIPTION
**Pull request by alex-s168**
(identical to #896 but removed the other git author (which was me))

The values in the config should be adjusted a bit

### CODE REVIEW

File: common/src/main/java/org/valkyrienskies/mod/mixin/server/MixinMinecraftServer.java [hunk 1]
```diff
@@ -229,6 +230,7 @@ private void postTick(final CallbackInfo ci) {
         // Only drag entities after we have updated the ship positions
         for (final ServerLevel level : getAllLevels()) {
             EntityDragger.INSTANCE.dragEntitiesWithShips(level.getAllEntities());
+            Weather2Compat.INSTANCE.tick(level);
```
Comment (StewStrong) [inline common/src/main/java/org/valkyrienskies/mod/mixin/server/MixinMinecraftServer.java:233]: Can you add an if statement here to only run Weather2Compat if the Weather2 mod is loaded?

**Comment by StewStrong** [comment]
Can you modify the code to only run if Weather2 is installed? Right now it seems like this will crash if someone uses VS2 without Weather2.

**Comment by alex-s168** [comment]
There is a `if (!LoadedMods.weather2) return` in the function
I'm pretty sure that the jvm only loads classes (in this case weather2) when it is used in the code

**Comment by alex-s168** [comment]
@StewStrong moved it anyways because nicer code

**Comment by StewStrong** [comment]
Nice! :D

**Pull request merged**